sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -16,7 +16,7 @@ while making it faster to get up and running with batch, streaming, and interact
 Install the Azure Storage Files Data Lake client library for .NET with [NuGet][nuget]:
 
 ```Powershell
-dotnet add package Azure.Storage.Files.DataLake --version 12.0.0-preview.5
+dotnet add package Azure.Storage.Files.DataLake --version 12.0.0-preview.6
```

Comment:
@AlexGhiondea  @tg-msft  do we expect the version numbers in the readme files to be a common thing across the .NET Repo? I only ask because at this point it has been a pretty sore spot. We cannot figure out the correct version to include in them to enable the various scenario (i.e. consumers coming to the repo vs devs working on the code to make it match what is under development).

Reply:
NuGet doesn't let us install the latest Preview package so we need these preview versions across the repo.  We don't have this for [packages that are GA](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs#install-the-package).  I think the README should be optimized for consumers and always reference the latest release on NuGet.

Reply:
>NuGet doesn't let us install the latest Preview package so we need these preview versions across the repo. We don't have this for packages that are GA.

That is a nice generic way to do it

> I think the README should be optimized for consumers and always reference the latest release on NuGet.

That is definitely a point of high debate. If you do optimize them for consumers that means they can only ever have samples and list features for things that are released which means they will quickly become old and stale and not ever demonstrate the new features.